libs/blocks/media/media.css

```diff
@@ -166,7 +166,6 @@ div[class*="-up"] .media .foreground > .media-row {
 
 .media.qr-code .qr-button-container {
   display: inline;
```

Comment:
Wouldn't `inline-block` be better suited here? It would also apply the same margins applied to other elements in the section. Overall this looks like it can use a bit more attention, as on RTL the icon positioning is also wrong:
<img width="388" alt="Screenshot 2024-10-22 at 14 28 11" src="https://github.com/user-attachments/assets/8130bead-a614-4dc8-9a4a-b58365ac0322">